bahmni-proxy/resources/unauthorized.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Down For Maintenance</title>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+		<link rel="stylesheet" href="style.css" />
+	</head>
+	<body>
+		<img src="/bahmni-logo.png" alt="Bahmni" width="200" />
+		<h1>Unauthorized Access</h1>
+		<p>
+			Sorry for the inconvenience, we understand that this could be impacting
+			patient care.
+			<br />
+			Please click <a href="/bahmni/home">here</a> to go back to the dashboard
```

Comment:
This copy text is not correct for Unauthorized Access

Reply:
It Could be

`You don't have the correct permissions to perform this action. For more information contact your system administrator.`

Reply:
and then you can have the `please click bla bla to go back...`

Reply:
changed the copy text